test/info/info-help.test.js

```diff
@@ -28,13 +28,6 @@ describe('should print help for info command', () => {
         expect(stdout).toContain(descriptionText);
         expect(stderr).toHaveLength(0);
     });
-    it('should output help even if other flags are supplied with help', () => {
-        const { stdout, stderr } = runInfo(['info', '--help', '--system']);
-        expect(stdout).toContain(usageText);
-        expect(stdout).toContain(descriptionText);
-        expect(stdout).not.toContain('System:');
-        expect(stderr).toHaveLength(0);
-    });
 
     it('should respect the --color=false flag', () => {
         const { stdout, stderr } = runInfo(['info', 'help', '--color=false']);
```

Comment:
Please improve tests, we should check what it is included `npm`/`yarn`/`webpack`/etc strings

Reply:
Not only for that test

Reply:
This tests is for help flag used with `info`. But will add these in the tests for `info` output.